src/server/req_manager.c

```diff
@@ -3923,7 +3923,7 @@ check_resource_set_on_jobs_or_resvs(struct batch_request *preq, resource_def *pr
 static void
 timed_send_rescdef()
 {
-	send_rescdef(0);
+	send_rescdef(1);
```

Comment:
Might want to add a comment here indicating why the update is being forced.

Reply:
added it in commit https://github.com/PBSPro/pbspro/pull/1452/commits/8f309272ad00f7587c830cad07238cb40598e24a